src/net/sourceforge/kolmafia/session/ResultProcessor.java

```diff
@@ -2562,11 +2568,11 @@ private static void gainItem(boolean adventureResults, AdventureResult result) {
         break;
 
       case ItemPool.MERKIN_LOCKKEY:
-        MonsterData monster = MonsterStatusTracker.getLastMonster();
-        if (monster == null) {
+        MonsterData merkin_monster = MonsterStatusTracker.getLastMonster();
```

Comment:
Our naming convention would be merkinMonster, not merkin_monster

Reply:
got it.